qemu/tests/cfg/rng_bat.cfg

```diff
@@ -24,3 +24,8 @@
         driver_verifier_cmd = "cat /sys/devices/virtual/misc/hw_random/rng_current"
         driver_name = virtio
         rng_data_rex = "\w+"
+        check_rngd_service = "systemctl status rngd"
+        start_rngd_service = "systemctl start rngd && rngd -r /dev/hwrng -o /dev/random"
```

Comment:
Can you please explain why it should have `rngd -r /dev/hwrng -o /dev/random` even though had run `systemctl start rngd` first?

Reply:
After starting rngd service, "rngd -r /dev/hwrng -o /dev/random" will hook up the virtio-rng hwrng to the guest's entropy pool.

Reply:
There will be a rng daemon started by the rngd service, and it will redirect the entropy which generated by `/dev/hwrng` to the entropy pool, you can verify it by running `lsof /dev/hwrng`.

So here you just need either of the commands.

Reply:
Updated, Thanks! @luckyh